common/filespec_test.go

```diff
@@ -122,12 +122,12 @@ var testCasesCopy []*testCaseCopy = []*testCaseCopy{
     &testCaseCopy{
         FileSpec{Type: "path", Path: "_tests/files"},
         false,
-        "6481ba58a09ec62ff50325df54cf8966",
+        "3ec09caa093ac5b9211001adf722a418",
```

Comment:
Instead of changing the hash here, what about making a subdirectory in files ("filespec_test"?) and putting some file in it (it could be a copy of a.txt with a new name)?
Then, we won't have to update this again if we need to add more test files.
We would still have to recompute this specific hash, but we should be protected later.